tests/test_save.py

```diff
@@ -127,5 +129,5 @@ def test_model_manipulation(model_class, storage_method):
         del model, env
 
     finally:
-        if os.path.exists("./test_model"):
-            os.remove("./test_model")
+        if os.path.exists(model_fname):
+            os.remove(model_fname)
```

Comment:
Shoud be `'{}.pkl'.format(model_fname)` no ?

Reply:
Good catch (I think this was a problem before this PR too). I've now changed `model_fname` everywhere to end in `.pkl` (`model.save` will not add a `.pkl` if this already exists, so the filename can be consistent throughout.)